XNAControls/XNAListBoxItem.cs

```diff
@@ -97,7 +98,19 @@ public float Alpha
     /// Whether this list box item is visible.
     /// Invisible list box items are not drawn and cannot be selected.
     /// </summary>
-    public bool Visible { get; set; } = true;
+    private bool visible = true;
+    public bool Visible
+    {
+        get { return visible; }
+        set
+        {
+            if (visible != value)
+            {
+                visible = value;
+                VisibilityChanged?.Invoke(this, EventArgs.Empty);
+            }
+        }
+    }
```

Comment:
Underscore fields that only exist to back properties.

```suggestion
    private bool _visible = true;
    public bool Visible
    {
        get { return _visible; }
        set
        {
            if (_visible != value)
            {
                _visible = value;
                VisibilityChanged?.Invoke(this, EventArgs.Empty);
            }
        }
    }
```